lib/TodoList.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'model.dart';
+
+class TodoList extends StatelessWidget {
+  final List<TodoTasks> list;
+
+  TodoList(this.list);
```

Comment:
Vet inte om du behöver detta då vi har lite olika kod men när jag ändrade namn på min lista i "model.dart" så var jag även tvungen att ändra i min todolist. 

Jag ser att din lista i model nu heter _task och då kanske du även måste byta namn här? Jag vet inte om du måste göra det men jag var tvungen att göra det. Alltså: 
```
class TodoList extends StatelessWidget {
  final List<TodoTasks> _task;

```
  TodoList(this._task);

och 

```
 Widget build(BuildContext context) {
    return ListView(
        children: _task.map((task) => _taskItem(context, task)).toList());
  }
```

Jag ändrade detta för att det blev rött i koden för mig att ha kvar det gamla namnet jag hade på listan men eftersom våra koder är uppbyggda olika så vet jag inte om det här är något du behöver!